react/table/src/auth/KeycloakViews.js

```diff
@@ -0,0 +1,9 @@
+export const AuthenticatedView = ({ children, keycloak }) => {
+  const authenticated = keycloak.initialized && keycloak.authenticated;
```

Comment:
it is always passed from CustomElement component as
```
  (window &&
    window.entando &&
    window.entando.keycloak && { ...window.entando.keycloak, initialized: true }) || {
    initialized: false,
```
so it will always have the `initialized` value in it